### ISSUE TITLE
V2 : Feature Request : More Info per work item in an estimate session

### ISSUE THREAD

**Issue by RafaelKoch** [cschleiden/azure-boards-estimate#34]
Right now we can see the description of a work item. This is often not enough to decide what to vote. This would be helpful:

- show also the acceptance criteria
- show the parent tree (especially needed when working with multiple epics having multiple features - developers frequently ask "to what does this belong?". So You end up opening the work item, which shows the feature... which You open to see the parent epic)

----
How do we work in V1? -> select from a query and start an estimate. In this query include epics and features together with the PBI/Bugs. Then at least the estimate list contains those in order and we are spared clicking too much. As for the Info on the PBI/Bug itselt: no solution but opening it in a new tab.
So basically V2 looks like a step into the right direction, being totally awesome if this suggestion finds Your fancy :-)

**Comment by bryder08** [comment]
Perhaps it would be useful to have options to show/hide the acceptance criteria and any other info affecting the layout?

**Comment by mnbf9rca** [comment]
just being able to click on teh item so it opens in a new tab woudl be a good start

**Comment by cschleiden** [comment]
You can click on the Id + Work Item Type to open in a dialog/new tab:

![image](https://user-images.githubusercontent.com/2201819/57706802-4aa48000-761b-11e9-96f5-b50c45672ebd.png)

**Comment by RafaelKoch** [comment]
I know I can open a new Tab. That is what on V1 we did all the time. Now on V2 we still do because acceptance or repro is not shown.

The new layout is a great idea and much appreciated, so all the more desirable to follow through with it and get rid ov a navigation pain altogether :-)

**Comment by cschleiden** [comment]
This is difficult since work item types could have any shape or number of fields. For now the logic is to look at the `Details` page of a work item type and take the first HTML field on this page.

**Comment by cschleiden** [comment]
We could extend this to look at all HTML fields on `Details`...

**Comment by RafaelKoch** [comment]
Just had a quick bug estimate with the team today... had to open half of the bugs to see Acceptance Data. 

And I have to add: Comments. We would also need the comments.

I am really not into TypeScript and the inner workings of Azure DevOps extensions, so no idea if I am really runing Your mood here. Sorry for that :-)

**Comment by mhalkyer** [comment]
This would be a great addition. Perhaps a settings area where you can set which fields to show for each Work Item Type.
